cfpb/cfgov-refresh
# Mega Menu: Remove h2 heading wrapper around trigger button

Seems kind of weird that we wrap the hamburger button in an `h2` and reset the default styles for `h2`. I think this was probably originally added in anticipation of more content being in this part of the menu that didn't show up.

**I also wondered if there's an accessibility reason to have heading content in a `nav`, but it seems to work the same with or without with voiceover. Anyone else know of any reason the heading wrapper may be needed?**

## Removals

- Remove h2 `o-mega-menu_heading` heading wrapper around mobile trigger button.

## Changes

- Adds h2 heading wrapper around top-level menu links.

## Testing

1. Pull branch and build.
2. Visit the mobile version of the site and expand/collapse the hamburger menu. It should work just like production.
